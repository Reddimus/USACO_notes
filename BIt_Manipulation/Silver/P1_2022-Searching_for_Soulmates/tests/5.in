10
1224022719 1134817430
626328603 46765972
68623321 2560305025
1818760402 2618197700
2838799127 4052981191
1524286819 1956570503
820253683 817674057
1263857005 363349394
3837034299 2847276109
194591977 365439781
