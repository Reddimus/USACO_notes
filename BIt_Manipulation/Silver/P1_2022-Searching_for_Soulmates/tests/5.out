84
75
72
86
90
85
87
72
89
77
